src/components/Integrations/applicationsList/ApplicationsList.vue

```diff
@@ -0,0 +1,366 @@
+<template>
+  <bf-page class="applications-list">
+    <bf-stage slot="stage" element-loading-background="transparent">
+      <div class="addIntegrationContainer">
+        <div class="description">
+          <p class="mb-16">
+            Applications support actions on various entities on the platform
+            such as "Files", "Records", and "Datasets". Registered applications
+            can be triggered from the action-menu associated with the targeted
+            entities.
+
+            <a
+              href="https://docs.pennsieve.io/docs/preventing-files-from-being-included-during-publishing"
+              target="_blank"
+            >
+              What's this?
+            </a>
+          </p>
+        </div>
+
+        <div class="reg-button">
+          <bf-button v-if="hasAdminRights" @click="openAddIntegration">
+            Register Application
```

Comment:
When I click on "Register Application", I still see webhook references, for example some api URL suggestions.
We also still have the generic "Add Integration" titles/text there as well. Do we plan to keep it that way?

Reply:
@edmore Thanks for noticing this - let me fix this before we deploy this code to prod.